apps/backend/src/users/user.entity.ts

```diff
@@ -18,7 +19,7 @@ export class User {
   email: string;
```

Comment:
💭 Might be useful to add some validators here both here and on line 25 for linkedIn url
```suggestion
\@isEmail()
  email: string;
```